scripts/benchmark.py

```
@@ -0,0 +1,74 @@
+
+print()
+print("=" * 75)
+print(f"{'pvalue_npy benchmark':^75}")
+print("=" * 75)
+
+for n in SIZES:
+    a, b, c, d = _random_npy_arrays(n)
+
+    label, t = bench(
+        f"fisher.pvalue_npy          (n={n:>7,})",
+        lambda: fisher.pvalue_npy(a, b, c, d),
```

Ruff (0.16.1)

[warning] Function definition does not bind loop variable `a`

(B023)